src/main/resources/reference.conf

```diff
@@ -42,9 +42,13 @@ conseil {
 }
 
 metering-api {
+  protocol: "http"
   host: "localhost"
   port: 8080
   read-timeout: 15000 milliseconds
+  gather-interval: 15 minutes
+  stats-interval: 5 minutes
```

Comment:
I use it as a reminder that we use 5m routes -  I did not want to hardcode the `period_start` of the entry as from the influx we get only `period_end` so I subtract the 5minutes which I get from config. In the future it may change.